WalletWasabi.Fluent/ViewModels/ApplicationViewModel.cs

```diff
@@ -38,6 +39,22 @@ public ApplicationViewModel()
 
 		ShowCommand = ReactiveCommand.Create(() => ShowRequested?.Invoke(this, EventArgs.Empty));
 
+		AboutCommand = ReactiveCommand.Create(  () =>
+		{
+			var dialogScreen = MainViewModel.Instance?.DialogScreen;
+
+			if (dialogScreen is null)
+			{
+				return;
+			}
+
+			// Don't navigate to the about page if there's a currently open
+			// dialog.
```

Comment:
Why?

Reply:
Because the real dialog operations should take precedence over a mere about dialog. Tis relatively easy to misclick them in macOS.

Reply:
Ok, but the user will be confused if they click on the button but nothing happens.

Reply:
Yeah im thinking of disabling it when there's an active dialog.. or overlay it with the secondary dialog but that's ugly  i think

Reply:
Yeah adding a `CanExecute` makes sense to me.

btw how about... adding CanExecute to disable the button only when `dialogScreen.CurrentPage.IsBusy` is true?
And add a navigate back button to the AboutScreen. So basically the user can open it anytime, even if another dialog is opened and they can also navigate back to the previous dialog. Does it worth the effort?

Reply:
> And add a navigate back button to the AboutScreen. So basically the user can open it anytime, even if another dialog is opened and they can also navigate back to the previous dialog. Does it worth the effort?

Nope i dont think it is worth spending more time on this with that

> btw how about... adding CanExecute to disable the button only when dialogScreen.CurrentPage.IsBusy is true?
Hmm... i dont like to add uncertainty so let's just go with CanExecute but with dialogScreen.CurrentPage is null